docs/user-guide/configuration.rst

```diff
@@ -171,6 +171,47 @@ Here is a full example showing how to configure all possible sinks:
         api_key: "datadog api key"
         default: false
 
+Configuration secrets
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Some of the configuration values are considered secrets, and cannot be saved in plain text format.
+We recommend using `SealedSecrets <https://github.com/bitnami-labs/sealed-secrets>`_
+or one of the other secret management system for Kubernetes, to encrypt the secret values.
+
+As an alternative, we can pull secret values from Kubernetes secrets.
+
+Here's how:
```

Comment:
I think this would be easier to understand if you reversed the order: first show the secret, then create the env variable, then pull the value into the string. (It doesn't make sense to me to explain first how you pull it in from an environment variable before you've explained why it is an environment variable.)

I would also remove the text "Here's how" as it is obvious, but that's just a personal preference

Reply:
sure